Python.gitignore

```diff
@@ -52,7 +52,6 @@ coverage.xml
 
 # Translations
 *.mo
-*.pot
```

Comment:
This rule has been largely unchanged for 7+ years, so I have several concerns about dropping this:

 - Most `.pot` files are generated, right? Would this be noisy to now have all `.pot` files in version control?
 - For situations where include/exclude files from version control is situational, we like to have a comment with context about when and why you should care about this rule. Here's an example:

https://github.com/github/gitignore/blob/218a941be92679ce67d0484547e3e142b2f5f6f0/Rails.gitignore#L24-L25

Can we do that for this case, to explain for Plone usage what they should be aware of?
 
 - if we're trying to exclude a specific file by convention, could we keep this rule but add a `!manual.pot` or `!lib/manual.pot` rule to opt-in for this specific file?